addons/mrp/tests/test_replenish.py

```diff
@@ -72,6 +72,49 @@ def test_mrp_orderpoint_leadtime(self):
         # for manufacturing delay should be taken from the bom
         self.assertEqual("4.0 days", info.wh_replenishment_option_ids.lead_time)
 
+    def test_rr_picking_type_id(self):
+        """Check manufacturing order take bom according to picking type of the rule triggered by an
+        orderpoint."""
+
+        self.product_4.route_ids = self.env.ref('mrp.route_warehouse0_manufacture')
+        self.picking_type_2 = self.warehouse_1.manu_type_id.copy({'sequence': 100})
```

Comment:
```suggestion
        picking_type_2 = self.warehouse_1.manu_type_id.copy({'sequence': 100})
```
No need to put it in `self`.